samples/subsys/power/stm32_pm/src/main.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2016 Intel Corporation.
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sample.h"
+
+struct device *gpio_bn, *gpio_led, *gpio_led2;
+
+bool ready_to_sleep;
+
+/* in msec */
+#define SLEEP_TIME 3000
+
+/* setting to K_TICKS_FOREVER willl activate the deepsleep mode */
+
+#define DEMO_DESCRIPTION	\
```

Comment:
This is nice, but there should be a `README.rst` that documents this sample and is integrated into the built documentation, as there is for most other samples.

Reply:
done